exercises/sending-signals-external/solution/starter.py

```diff
@@ -0,0 +1,37 @@
+import asyncio
+
+from shared import TASK_QUEUE_NAME, WORKFLOW_ID_PREFIX, FULFILLED_WORKFLOW_ID_PREFIX, create_pizza_order
+from temporalio.client import Client
+from workflow import PizzaOrderWorkflow
+from workflow import FulfillOrderWorkflow
+
+
+async def main():
+    # Create client connected to server at the given address
+    client = await Client.connect("localhost:7233", namespace="default")
+
+    order = create_pizza_order()
+
+    # Execute a workflow
+    handle = await client.start_workflow(
+        PizzaOrderWorkflow.order_pizza,
+        order,
+        id=WORKFLOW_ID_PREFIX + f"{order.order_number}",
+        task_queue=TASK_QUEUE_NAME,
+    )
+
+    # Execute a workflow
+    await client.start_workflow(
```

Comment:
```suggestion
    await client.execute_workflow(
```

Might as well wait, otherwise any failure is non-obvious